src/predict.py

```diff
@@ -26,14 +27,15 @@
 from flask import Flask, send_from_directory
 from flask import request
 import gcsfs
-from transformers import AutoModelForSeq2SeqLM
+from transformers import AutoModelForCausalLM, AutoModelForSeq2SeqLM
 from transformers import AutoTokenizer
 from transformers import GenerationConfig
 
 app = Flask(__name__, root_path=os.path.join(os.getcwd(), "app/"))
 FLAGS = flags.FLAGS
 
 flags.DEFINE_string("model_path", None, "Path of HF model to load.")
+flags.DEFINE_string("hf_autoclass", None, "Name of the Transformers autoclass to instantiate the model with. Default is 'AutoModelForSeq2SeqLM'")
```

Comment:
Fix docstring. There is no longer a string default, instead it cascades through CausalLM -> Seq2SeqLM before exiting.